doc/TRANSPORTCERTSSHA256.md

```diff
@@ -32,7 +32,7 @@ payload:
 ```
 
 ## Prerequisite 
-For extended token validation is that **public keys from connector itself and other connectors MUST be loaded into truststore.** Reason for this is that TRUEConnector will, during startup:
+For extended token validation is that **public keys from connector itself and other connectors MUST be loaded into truststore.** Reason for this is that  will, during startup:
```

Comment:
Reason for this is that **TRUE Connector** will, during startup:  -> TRUE Connector is missing

Reply:
Added TRUE Connector